cpu/cc26xx-cc13xx/rf-core/prop-mode.c

```diff
@@ -364,12 +365,20 @@ prop_div_radio_setup(void)
   uint32_t cmd_status;
   rfc_radioOp_t *cmd = (rfc_radioOp_t *)&smartrf_settings_cmd_prop_radio_div_setup;
 
+  rf_switch_select_path(RF_SWITCH_PATH_SUBGHZ);
```

Comment:
Do we also need to set this in `soft_on_prop`? `rf-ble` calls `rf_core_primary_mode_restore` when it's done, and presumably we want that to switch the antenna over?